omx/gstomxaudiodec.c

```diff
@@ -423,6 +415,9 @@ gst_omx_audio_dec_loop (GstOMXAudioDec * self)
       gst_audio_get_channel_reorder_map (pcm_param.nChannels, self->position,
           omx_position, self->reorder_map);
 
+  GST_DEBUG_OBJECT (self, "PCM params: eNumData %d, eEndian %d, nBitPerSample %d",
+    pcm_param.eNumData, pcm_param.eEndian, pcm_param.nBitPerSample);
```

Comment:
Cast the arguments with type 'OMX_U32' to 'int' to avoid having compilation warnings.

Reply:
66cee90a7d9904dfc8155591da324492d4f520ef